CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id 'java'
+}
+
+group 'com.couchbase.mobiletestkit'
+
+ext {
+    DEF_VERSION = "2.7.0-56"
```

Comment:
Couple things here:
- There's probably no reason to define DEF_VERSION.  It will only be used as the default, for the version, below, right?  Probably clearer just to put it there.
- Most of these products get their version from a file called "version.txt".  I can see from the format of this file that you've been looking at the Java/Android build files.  They can serve as a guide for how to do that.
- Mostly we've been abbreviating COUCHBASE_LITE_ simply as CBL_.  EIther way....

Reply:
Regarding to the version, DEF_VERSION is only designed for people work on their locally environment and import cblite by gradle. I have a build job now and it publishes testserver app, the build script set the version and build number. Here are the links for the job http://mobile.jenkins.couchbase.com/view/CBL%202.x/job/couchbase-lite-java-desktop-testapp/ and the link for artifactory http://latestbuilds.service.couchbase.com/builds/latestbuilds/couchbase-lite-java/2.7.0/56/

Reply:
Ok.  Just a suggestion...